pkg/systemd/services/timer-dialog.jsx

```diff
@@ -92,13 +92,13 @@ const CreateTimerDialogBody = ({ owner }) => {
                     is24Hour
                     isOpen={repeatPatterns[idx].isOpen || false}
                     setIsOpen={isOpen => {
-                        const arr = [...repeatPatterns];
+                        const arr = JSON.parse(JSON.stringify(repeatPatterns));
```

Comment:
Heh, this must be the most expensive way ever to create a deep copy :grin: (But it doesn't matter here, convenience and robustness beats performance here)

Reply:
Yeah, but since it is maximum two elements object it will be just fast enough